SETUP/tests/manual_web/qunit.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
do we want to duplicate this file? We can move the other one and update the npm run script if we want the location here.

Reply:
This is just a hack as above.